crates/sui-types/src/randomness_state.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use move_core_types::{account_address::AccountAddress, ident_str, identifier::IdentStr};
+use serde::{Deserialize, Serialize};
+
+use crate::base_types::SequenceNumber;
+use crate::dynamic_field::get_dynamic_field_from_store;
+use crate::error::{SuiError, SuiResult};
+use crate::object::Owner;
+use crate::storage::ObjectStore;
+use crate::{id::UID, SUI_FRAMEWORK_ADDRESS, SUI_RANDOMNESS_STATE_OBJECT_ID};
+
+pub const RANDOMNESS_MODULE_NAME: &IdentStr = ident_str!("random");
+pub const RANDOMNESS_STATE_STRUCT_NAME: &IdentStr = ident_str!("Random");
+pub const RANDOMNESS_STATE_UPDATE_FUNCTION_NAME: &IdentStr = ident_str!("update_randomness_state");
+pub const RANDOMNESS_STATE_CREATE_FUNCTION_NAME: &IdentStr = ident_str!("create");
+pub const RESOLVED_SUI_RANDOMNESS_STATE: (&AccountAddress, &IdentStr, &IdentStr) = (
+    &SUI_FRAMEWORK_ADDRESS,
+    RANDOMNESS_MODULE_NAME,
+    RANDOMNESS_STATE_STRUCT_NAME,
+);
+
+/// Current latest version of the randomness state object.
```

Comment:
can you clarify that this is the semantic version of the inner object, not the sui object version?

Reply:
even better, removed